Core/clim-core/bordered-output.lisp

```diff
@@ -456,8 +456,29 @@
                           (draw-line* stream left bottom right bottom
                                       :ink ink
                                       :line-style line-style)))
-                       (updating-output-record  nil)
-                       (compound-output-record  (fn child))))))
+                       (updating-output-record nil)
+                       (compound-output-record (fn child))))))
+      (fn record))))
+
+(define-border-type :crossout (stream record
```

Comment:
This is often called "strikethrough". Is there a specific reason for going with "crossout"?

Reply:
I've checked names and I had two choices with the same meaning: `crossout` and `strikethrough`, I've picked the shorter one.